test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageControllerSpec.scala

```diff
@@ -3710,6 +3711,7 @@ class MultipleDisposalsTriageControllerSpec
         }
 
       }
+
```

Comment:
it'd be good to get some unit tests for the new behaviour in the `MultipleDisposalstriageController`